### DISCUSSION TITLE
建议脚本确定好CDN

### DISCUSSION THREAD

**Discussion by Workear34** [the1812/Bilibili-Evolved#3666] [功能建议 / Ideas]
目前raw.githubusercontent.com还是不行（反正我是连接不上）
cdn.jsdelivr.net目前已经恢复了，但因为是国外节点，速度可能会很慢
我推荐的CDN是UNPKG，而且脚本所用的库UNPKG都有
https://unpkg.com/vue@3.2.38/dist/vue.global.js
https://unpkg.com/jszip@3.10.1/lib/index.js
https://unpkg.com/@popperjs/core@2.11.6/dist/umd/popper.min.js
https://unpkg.com/tippy.js@6
https://unpkg.com/sortable@2.0.0/index.js
https://unpkg.com/color@4.2.3/index.js
https://unpkg.com/lodash@4.17.21/lodash.js
https://unpkg.com/marked@4.1.0/lib/marked.umd.js
https://unpkg.com/mdi@2.2.43/css/materialdesignicons.min.css

**Comment by the1812** [comment]
unpkg 能访问 github 仓库的文件吗 (脚本本体)

**Reply by Workear34** [reply]
貌似不能，但需要引用仓库本体的文件可以用fastly.jsdelivr.net/gh
例如这是我github pages的博客的CDN：https://fastly.jsdelivr.net/gh/workear34/workear34.github.io@main/
或者你可以看一下issues#3657，里面也有推荐

**Reply by FoundTheWOUT** [reply]
可以考虑发一下包，这样 unpkg 就能拿到脚本本体了
